src/main/java/io/sentry/samples/example/Application.java

```diff
@@ -120,8 +102,8 @@ public String UnhandledError() {
 	} 
 
 	public static void main(String[] args) {
-		inventory.put("wrench", 0);
-		inventory.put("nails", 0);
+		inventory.put("wrench", 1);
```

Comment:
Keep these consistent, or else you have to remember "which tool do I need to purchase 1 item vs +2 items in order to generate the error..."